### DISCUSSION TITLE
General Q

### DISCUSSION THREAD

**Discussion by v1gnesh** [DataEngineeringLabs/arrow2-convert#116] [General]
Hello,

Thank you for sharing your excellent, excellent work.

When you find time, can you please update the `complex_example` with examples showing the latest capabilities?

Any guidane on the GAT refactor and adding support for things like:
```rust
pub struct Numeric<const N: usize>(
    #[deku(map = |num: [u8; N]| LittleEndian::read_int(&num, N))] pub(crate) i64,
);
```

Any guidance on surfacing the `transparent` capability?
https://github.com/DataEngineeringLabs/arrow2-convert/issues/81

Thanks in advance!

**Comment by ncpenke** [comment]
Thank you @v1gnesh. GAT refactor is 90% complete, but I've been busy the last few months so haven't had a chance to finish. Hoping to wrap it up in the upcoming weeks. The refactor touches a lot of the core macro logic, but can either implement transparent or update the ticket with guidance. Thanks!

**Reply by v1gnesh** [reply]
Ah, sorry. I used the word guidance like in finance / markets (any guidance on earnings, etc.); meaning, any guestimate on when it could be delivered. 
At the moment, this is too advanced for my skill level. I first need to figure out how to effectively apply what you've shared for my use case :)

**Reply by ncpenke** [reply]
No worries that makes sense! Honestly, was hoping to get the refactor done several months ago, but time has been scarce. Since there's some interest, will try to get this wrapped up in the next few weeks.

**Reply by v1gnesh** [reply]
It's hard not to look into the Arrow universe these days. And you are the guide :1st_place_medal:
